### DISCUSSION TITLE
Is there a way to add grass type from which straw is harvestable?

### DISCUSSION THREAD

**Discussion by ivxvm** [vectorwing/FarmersDelight#1018] [Q&A]
I want it to drop from `biomeswevegone:prairie_grass` and `biomeswevegone:tall_prairie_grass` for example. Unfortunately, I couldn't find a tag for this. I also tried adding datapack loot modifier like this, but it didn't work:
```json
{
    "type": "farmersdelight:add_item",
    "conditions": [
        {
            "condition": "minecraft:random_chance",
            "chance": 0.2
        },
        {
            "condition": "minecraft:match_tool",
            "predicate": {
                "tag": "farmersdelight:straw_harvesters"
            }
        },
        {
            "condition": "block_state_property",
            "block": "biomeswevegone:prairie_grass"
        }
    ],
    "item": "farmersdelight:straw"
}
```
I guess this will only work from the mod itself, not externally.

**Comment by vectorwing** [comment]
@ivxvm You have to add the name of your new file into `global_loot_modifiers.json`:
https://github.com/vectorwing/FarmersDelight/blob/1.20/src/main/resources/data/forge/loot_modifiers/global_loot_modifiers.json

This file acts like a tag .json, adding anything in it to the loot modifier loader. You can create your own and specify your entries, the same way as tags.
